src/link/link_info/bridge.rs

```diff
@@ -70,6 +70,32 @@ const IFLA_BR_VLAN_STATS_PER_PORT: u16 = 45;
 const IFLA_BR_MULTI_BOOLOPT: u16 = 46;
 const IFLA_BR_MCAST_QUERIER_STATE: u16 = 47;
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct BridgeId {
+    pub priority: u16,
+    pub address: [u8; 6],
+}
+
+impl BridgeId {
+    pub(crate) fn from_bytes(buffer: &[u8]) -> Result<Self, DecodeError> {
+        if buffer.len() != 8 {
+            return Err("BridgeId buffer must be 8 bytes long".into());
+        }
+
+        // priority is encoded in big endian:
+        // https://elixir.bootlin.com/linux/v6.6.9/source/net/bridge/br_netlink.c#L1633
```

Comment:
This URL might not valid in the future. If you want to refer kernel code, please use function name and file name like `net/bridge/br_netlik.c function abc()`

Reply:
The URL will be valid because I've chosen a specific kernel version, not "latest". Actually, if you specify file and function, that can change in the future if the specific line is moved to a different function, or the file and function might even be deleted.

If you prefer, I can put the line of code instead:
```
// priority is encoded in big endian (from kernel's net/bridge/br_netlink.c br_fill_info():
// u16 priority = (br->bridge_id.prio[0] << 8) | br->bridge_id.prio[1];
```

Reply:
I mean the web site might decommissioned